static/css/thing.css

```diff
@@ -179,3 +200,16 @@
   font-weight: bold;
 }
 
+.thing-title {
+  width: 100%;
+  text-align: center;
+  position: fixed;
+  top: 20px;
+}
+
+.thing-title-icon {
+  height: 40px;
```

Comment:
These should all be in rem, I also think that 40px is slightly off for vertical alignment. For example, 
<img width="211" alt="screen shot 2018-01-19 at 6 59 11 pm" src="https://user-images.githubusercontent.com/2776089/35177102-2415da00-fd4b-11e7-9b8d-34ced96e7824.png">